### ISSUE TITLE
XOutput 3.32 is reporting as 3.31 (forces there is a new version message)

### ISSUE THREAD

**Issue by GoodSoulGermany** [csutorasa/XOutput#377]
Hey, just a minor thing but still annoys me. The latest version of XOuput (3.32) reports as 3.31. That forces the "There is a new version" message everytime I start the programm. Can you please exchange the .exe file with a correct version info? 

Thanks for this wonderfull programm. :-)

**Comment by Mubashir78** [comment]
Thanks @Fuj- ! But I really hope that the dev notices it and provides us with the latest version. He said he had added an auto refresh feature. I have also opened an issue hoping that the dev will fix it.

**Comment by kotenok2000** [comment]
When I compare 3.31 and 3.32 binaries in winmerge they are different.

**Comment by Fuj-** [comment]
You're right, hashes are different. It's likely 3.32 but reports as 3.31 because they forgot to update version info.
